package.json

```diff
@@ -10,7 +10,11 @@
     "prepare": "husky"
   },
   "dependencies": {
+    "@radix-ui/react-dialog": "^1.1.1",
+    "@radix-ui/react-icons": "^1.3.0",
+    "@radix-ui/react-slot": "^1.1.0",
     "@tanstack/react-virtual": "^3.8.4",
+    "class-variance-authority": "^0.7.0",
```

Comment:
ayoooo, I didn't know you could alias npm package names, we couuuld
```shell
npm i cva@npm:class-variance-authority
```
see: https://cva.style/docs/getting-started/installation

but in all seriousness, it doesn't hurt to add their settings for intellisense either:
```jsonc
{
  "tailwindCSS.experimental.classRegex": [
    ["cva\\(([^)]*)\\)", "[\"'`]([^\"'`]*).*?[\"'`]"],
    ["cx\\(([^)]*)\\)", "(?:'|\"|`)([^']*)(?:'|\"|`)"]
  ]
}
```
see: https://cva.style/docs/getting-started/installation#intellisense